src/main/scala/com/gu/contentapi/utils/PredicateUtils.scala

```diff
@@ -0,0 +1,20 @@
+package com.gu.contentapi.utils
+
+object PredicateUtils {
+
+  implicit class RichPredicate[A](val f: Function1[A, Boolean]) extends AnyVal {
+    def apply(v: A) = f(v)
+
+    def &&(g: Function1[A, Boolean]): Function1[A, Boolean] = {
```

Comment:
Just to add more that this is something not advanced at all, the [functional programming coursera 2 week course](https://www.coursera.org/learn/progfun1) ask you to implement similar boolean functions as an exercise when asking to add basic functions on set.